behat.yml.dist

```
@@ -0,0 +1,11 @@
+default:
+    suites:
+        default:
+            contexts:
+                - App\Tests\Behat\RequestContext
+                - App\Tests\Behat\UserContext
```

On utilise cette config pour différentier des tests
https://behat.org/en/latest/user_guide/configuration/suites.html#suite-filters
```suggestion
        users:
            contexts:
                - App\Tests\Behat\RequestContext
                - App\Tests\Behat\UserContext
            filters:
                tags: @user
```